backend/database/pictures/getRecentByNumber.js

```diff
@@ -2,13 +2,12 @@ const Sequelize = require('sequelize');
 const db = require('../connection');
 
 // Sequel Query
-const RENAME_THIS_QUERY = ``;
+const GET_RECENT_BY_NUMBER = `SELECT * FROM pictures ORDER BY createdAT DESC LIMIT ?, 1`;
```

Comment:
As I'm sure you know your query is correct, I made two new seed dates for Pictures #4 & #5 in the .env to make sure I was getting Picture # 5 back. 

So there's two ways I'm thinking about this. We either
A) Don't allow users to upload multiple photos at once ( which would result in the same createdAt times potentially ). This might solve our problem, but will be tedious for uploader.

B) We can ORDER BY the picture's id attribute, assuming that all ids will be in ascending order and unique.

I'm fine with either approaches, @j-palos please make an executive decision.

Reply:
Also jw what the `,1` is for ? Was that to test ascending and descending ?

Reply:
> As I'm sure you know your query is correct, I made two new seed dates for Pictures #4 & #5 in the .env to make sure I was getting Picture # 5 back.
> 
> So there's two ways I'm thinking about this. We either
> A) Don't allow users to upload multiple photos at once ( which would result in the same createdAt times potentially ). This might solve our problem, but will be tedious for uploader.
> 
> B) We can ORDER BY the picture's id attribute, assuming that all ids will be in ascending order and unique.
> 
> I'm fine with either approaches, @j-palos please make an executive decision.

I'm okay with option A. The chance is very small and the consequences aren't particularly disastrous. 
When uploading images from an api call, it shouldn't be an issue either.

Reply:
> Also jw what the `,1` is for ? Was that to test ascending and descending ?

It's to keep it limited to the one matching result, yes. The n , x indicates a range between n and n+x

Reply:
Sounds reasonable to me.

I can manipulate the seed data to add one and two more seconds to the 4th and 5th pictures so that anyone else testing off the seed data will pass tests. As of now I'm comfortable with @ajmrquez 's code completely.